### DISCUSSION TITLE
Cart total amount is the same as subtotal amount which is wrong

### DISCUSSION THREAD

**Discussion by SaeedYasin** [Shopify/function-examples#195] [Feedback: Delivery Customization API]
Here is what is shown on the checkout during function execution,
<img width="473" alt="Screenshot 2023-03-28 at 5 13 58 PM" src="https://user-images.githubusercontent.com/26478201/228188962-08fb1fde-bf14-4458-ba31-69249a02a403.png">

Here is what the function receives as an input,
<img width="950" alt="Screenshot 2023-03-28 at 5 13 15 PM" src="https://user-images.githubusercontent.com/26478201/228188827-22fb14ed-024d-4b30-8f7b-84ca7a1b5000.png">

The value for total and subtotal is the same, which is not what the checkout is showing. Any solution for this?

**Comment by SaeedYasin** [comment]
This only happens in delivery customizations, but works fine in payment customizations.

**Comment by aldoalprak** [comment]
This bug still exists. Any update on this?
